Iterable-iOS-SDK/IterableAPI.m

```diff
@@ -223,18 +223,39 @@ - (void)sendRequest:(NSURLRequest *)request onSuccess:(void (^)(NSDictionary *))
                                                                    NSURLResponse *response,
                                                                    NSError *error)
     {
-        if ([data length] > 0 && error == nil) {
-            error = nil;
-            id object = [NSJSONSerialization
-                         JSONObjectWithData:data
-                         options:0
-                         error:&error];
-            if(error) {
+        NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *)response;
```

Comment:
is it possible for `response` to not be an `NSHTTPURLResponse`?

Reply:
It will be `NSHTTPURLResponse` for any HTTP response.
Apple docs:
`
Whenever you make HTTP URL load requests, any response objects you get back from the NSURLSession, NSURLConnection, or NSURLDownload class are instances of the NSHTTPURLResponse class.
`